core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller.rb

```diff
@@ -97,7 +97,9 @@ def update
   # @response [object<errors: object<title: string>>] 404 Not found
   # @response 401 Not authorized
   def deploy_containers
-    deployment = resource_project.deployments.active.find(params[:id])
+    deployment = resource_project.deployments.find(params[:id])
```

Comment:
In `UffizziCore::DeploymentRepo` there is scope `existed`. I think we can use it here.